continuous_training/composer/labs/chicago_taxifare/setup.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages
+from setuptools import setup
+
+REQUIRED_PACKAGES = [
+]
```

Comment:
Is there TF versions and / or Airflow versions to pin this too?

Reply:
Since we're going to be submitting this package to CAIP for training, we don't need to worry about an Airflow version. Strictly speaking, if we wanted to be careful we could include 'tensorflow==2.1.0', but we can also specify the version of TF when submitting the training job. What do you think?

Reply:
Got it, makes sense to me now to keep as is :)